<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DFADialog</class>
 <widget class="QDialog" name="DFADialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="stateNameLine">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="StartCheckBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>85</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Start State?</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="AcceptCheckBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Accept State</string>
   </property>
  </widget>
  <widget class="QPushButton" name="EnterStateBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="stateCounterLbl">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>0</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="statesLbl">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>74</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Total States:</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="symbolLine">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="EnterSymBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="symLbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Total symbols:</string>
    </property>
   </widget>
   <widget class="QLabel" name="numSymLbl">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>321</width>
     <height>101</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="StateLbl1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>State: </string>
    </property>
   </widget>
   <widget class="QLabel" name="currStateLbl">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="SymbolLbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>On Symbol: </string>
    </property>
   </widget>
   <widget class="QLabel" name="OnSymbolLbl">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="goesToLbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Goes To:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="gotoLine">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="EnterTransBtn">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter</string>
    </property>
   </widget>
   <widget class="QLabel" name="numTransLbl">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Total Transitions: </string>
    </property>
   </widget>
   <widget class="QLabel" name="numForTransLbl">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="startStateErrorLbl">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Academy Engraved LET</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="transErrorLbl">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>270</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>373</x>
     <y>63</y>
     <width>221</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buildBtn">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>240</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Build</string>
   </property>
  </widget>
  <widget class="QLabel" name="buildErrBtn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>270</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGraphicsView" name="machineView">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>340</y>
     <width>500</width>
     <height>350</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="testLine">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>301</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="testBtn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>300</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
